pmd/rulesets/backend-java-ruleset.xml

```diff
@@ -33,7 +33,7 @@
         <properties>
             <!--Ignore AvoidHardcodedConnectionConfig on classes where the class name ends with IT or Test-->
             <property name="violationSuppressXPath"
-                      value="//ClassOrInterfaceDeclaration['^[A-Z][a-zA-Z0-9]*(Test|IT)(s|Case)?$']"/>
+                      value="//ClassOrInterfaceDeclaration['^[A-Z][a-zA-Z0-9]+(Test|IT)(s|Case)?$']"/>
```

Comment:
I don't think that `TestHelper` matches any of those regexes :thinking:

Reply:
indeed :sweat_smile: it was another rule (TestClassWithoutTestCases) I'll fix it